src/Raven.Server/Routing/RequestRouter.cs

```diff
@@ -375,12 +382,19 @@ private static void ThrowUnknownAuthStatus(RavenServer.AuthenticationStatus stat
             RavenServer.AuthenticateConnection feature,
             AuthorizationStatus authorizationStatus)
         {
+            TryGetUserAgent(context, out var userAgent);
+            var userAgentChrome = userAgent.Contains("chrome");
```

Comment:
Why do we care? We can simply always include this

Reply:
Note that the same message is releavnt for FireFox, Safari, etc

Reply:
good point! + we don't need this detection which is also fragile

Reply:
np
(this chrome detection was inserted only because that was the issue definition)